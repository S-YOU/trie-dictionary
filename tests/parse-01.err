nodes: 2
nodes: 3
nodes: 2
nodes: 0
