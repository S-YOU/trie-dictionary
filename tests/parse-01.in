insert helloworld
    inesrt                     janiemoge                        
             prev 0 3 6
  find hel
    find        helium                 
      Find    helium
   finda              blabla
           find     hell0
      delete 4294967297
        del3te 0
                  delete        18446744073709551616
   prev 18446744073709551616 36893488147419103232 73786976294838206466
     delete 0
                 find                         hel                                                                      

prev
1
0
1
delete
1
   pref 1 0 0
      brev 1 0 0 
  brew    1   0  0
                            insert hello_world
         delete -5
 remove 1
find
prev
delete
insert
                clean  
                clear  
  prev 0
    prev 0 1
       prev 0 1 144

