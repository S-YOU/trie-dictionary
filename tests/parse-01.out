word number: 0
ignored
word number: 1
YES
NO
ignored
ignored
ignored
ignored
ignored
ignored
ignored
deleted: 0
NO
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
cleared
ignored
ignored
ignored
ignored
